lib/remote/configstageshandler.cpp

```diff
@@ -97,6 +100,11 @@ void ConfigStagesHandler::HandlePost(
 {
 	namespace http = boost::beast::http;
 
+	if (m_RunningPackageUpdates) {
+		return HttpUtility::SendJsonError(response, params, 409,
```

Comment:
What about 423?

Reply:
Yes, 423 Locked sounds more appropriate to signal "please try again later", 409 Conflict could also be interpreted as "the config you sent introduced a conflict, fix it and try again".